project.clj

```diff
@@ -1,7 +1,7 @@
 (def schema-tools-version "0.9.0")
-(def schema-version "1.1.3")
+(def schema-version "1.1.4")
 
-(defproject threatgrid/ctim "0.4.23-SNAPSHOT"
+(defproject threatgrid/ctim "0.4.24-SNAPSHOT"
```

Comment:
It is still the `0.4.24-SNAPSHOT` version. The version is only changed after a new release.